modules/manage/pages/database-offline.adoc

```diff
@@ -19,58 +19,131 @@ include::ROOT:partial$_show_page_header_block.adoc[]
 
 == Introduction
 
-Sync Gateway enables a database to be taken offline and brought back online.
-This is done without stopping the Sync Gateway instance and without affecting other databases served by it.
+Sync Gateway enables a database to be taken offline and brought back online without stopping the Sync Gateway instance and without affecting other databases served by it.
 
 The change of status (online or offline) of a database occurs only in the specific Sync Gateway instance addressed.
 It is not reflected by other Sync Gateway instances using that database.
-To achieve that, you need to coordinate individual change operations in each of the required Sync Gateway instances.
+To affect multiple instances, you need to coordinate individual change operations in each of the required Sync Gateway instances.
 
+=== Use Cases
 
-== Use Cases
+You may need to take a database offline or bring it back online in the following scenarios:
 
-Specific uses for the database offline/online functionality include:
+* Taking a database offline without affecting other databases
+* Changing configuration properties for a database (while it is offline) without needing to restart Sync Gateway
+* xref:resync.adoc[Resynchronizing a database] while it is offline
+* Responding to a lost DCP or TAP feed (Sync Gateway does this automatically)
+* Creating a database in an offline state so that the start of service delivery for the database can be postponed or coordinated across Sync Gateway instances
+* Performing a Couchbase Server upgrade
```

Comment:
```suggestion
* You need to take a single database offline without affecting other databases.
* You need to change the configuration properties for a database without restarting Sync Gateway.
* You need to xref:resync.adoc[resynchronize a database] while it is offline.
* You are responding to a lost DCP or TAP feed.
Sync Gateway does this automatically.
* You created a database in an offline state, so that the start of service delivery for the database can be postponed or coordinated across Sync Gateway instances.
* You need to run a Couchbase Server upgrade.
```